src/main/java/com/muxiu1997/sharewhereiam/integration/journeymap/WaypointManager.java

```diff
@@ -0,0 +1,66 @@
+package com.muxiu1997.sharewhereiam.integration.journeymap;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import javax.annotation.Nullable;
+
+import net.minecraft.client.Minecraft;
+
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+import journeymap.client.model.Waypoint;
+
+@SideOnly(Side.CLIENT)
+public class WaypointManager {
+
+    public static WaypointManager INSTANCE = new WaypointManager();
```

Comment:
Is there any benefit to this being a singleton instance over fully static methods?